cmd/templates-api/muxer.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"net/http"
+	"regexp"
+
+	"github.com/dimfeld/httptreemux/v5"
+	goaHTTP "goa.design/goa/v3/http"
+)
+
+type muxer struct {
+	*httptreemux.ContextMux
+}
+
+// newMuxer returns a Muxer implementation with custom 404 not found error handler.
+func newMuxer() goaHTTP.Muxer {
+	r := httptreemux.NewContextMux()
+
+	r.EscapeAddedRoutes = true
+	r.NotFoundHandler = func(w http.ResponseWriter, req *http.Request) {
+		_ = writeError(req.Context(), w, NotFoundError{})
+	}
+	r.PanicHandler = func(w http.ResponseWriter, req *http.Request, value interface{}) {
+		_ = writeError(req.Context(), w, PanicError{Value: value})
+	}
```

Comment:
Aby sme vedeli pouzit rovnaky layout chyby aj pre notFound aj panic, tak sa musi spravit vlastny `muxer`.
Tieto 2 bloky su zmenene, inak je to default Goa implementacia.